src/Http/Http.Extensions/test/RequestDelegateGenerator/RequestDelegateGeneratorIncrementalityTests.cs

```diff
@@ -44,8 +45,12 @@ public async Task MapAction_ChangeReturnType_TriggersUpdate()
     [Fact]
     public async Task MapAction_ChangeBodyParamNullability_TriggersUpdate()
     {
-        var source = @"app.MapGet(""/"", ([FromBody] Todo todo) => TypedResults.Ok(todo));";
-        var updatedSource = @"app.MapGet(""/"", ([FromBody] Todo? todo) => TypedResults.Ok(todo));";
+        var source = $"""app.MapGet("/", ([{typeof(FromBodyAttribute)}] {typeof(Todo)} todo) => TypedResults.Ok(todo));""";
+        var updatedSource = $"""
+#pragma warning disable CS8622
```

Comment:
For my learning, why do we get this warning here?

Reply:
This is a result of the bug outlined in https://github.com/dotnet/aspnetcore/issues/46622